ocaml/runtime/custom.c

```diff
@@ -112,49 +115,12 @@ CAMLexport value caml_alloc_custom_mem(const struct custom_operations * ops,
                                        uintnat bsz,
                                        mlsize_t mem)
 {
-<<<<<<< HEAD
-
-  mlsize_t mem_minor =
-    mem < caml_custom_minor_max_bsz ? mem : caml_custom_minor_max_bsz;
-  mlsize_t max_major =
-    /* The major ratio is a percentage relative to the major heap size.
-       A complete GC cycle will be done every time 2/3 of that much memory
-       is allocated for blocks in the major heap.  Assuming constant
-       allocation and deallocation rates, this means there are at most
-       [M/100 * major-heap-size] bytes of floating garbage at any time.
-       The reason for a factor of 2/3 (or 1.5) is, roughly speaking, because
-       the major GC takes 1.5 cycles (previous cycle + marking phase) before
-       it starts to deallocate dead blocks allocated during the previous cycle.
-       [heap_size / 150] is really [heap_size * (2/3) / 100] (but faster). */
-    caml_heap_size(Caml_state->shared_heap) / 150 * caml_custom_major_ratio;
-  mlsize_t max_minor =
-    Bsize_wsize (Caml_state->minor_heap_wsz) / 100 * caml_custom_minor_ratio;
-  value v = alloc_custom_gen (ops, bsz, mem, max_major, mem_minor, max_minor);
   size_t mem_words = (mem + sizeof(value) - 1) / sizeof(value);
+  value v = alloc_custom_gen (ops, bsz, mem, 0, get_max_minor());
+  // XXX mshinwell for sdolan: please check this conflict carefully
+  // and also, see if it relates to the XXX in bigarray.c
```

Comment:
This looks fine, but I'd prefer if you put `mem_words` _after_ the call to `alloc_custom_gen`. (This matches how it is in trunk upstream, giving us one less line of diff for next time)